bin/s2nc.c

```diff
@@ -21,11 +21,9 @@
 #include <getopt.h>
 #include <fcntl.h>
 
-#include <openssl/crypto.h>
-#include <openssl/err.h>
-
 #include "api/s2n.h"
 #include "common.h"
+#include "crypto/s2n_fips.h"
```

Comment:
I'm not sure this entirely fixes our problem since the libs2n.{a,so} could have been linked differently from the s2nd/s2nc. I think instead we need to make `s2n_is_in_fips_mode()` part of the public api in `s2n.h`

Reply:
Totally- was kind of surprised it isn't public.

Reply:
Yeah I think it makes sense for it to be. Then we don't need to include this internal header either.